cmd/atlas/atlas.go

```diff
@@ -53,6 +62,74 @@ func initConfig() {
 	}
 }
 
+// createConfigFromMongoCLIConfig creates atlasCLI config file from mongoCLI config file.
+func createConfigFromMongoCLIConfig(err error) bool {
+	// search mongoCLI config only if atlasCLI config doesn't exist
+	var e viper.ConfigFileNotFoundError
+	if !errors.As(err, &e) {
+		return false
+	}
```

Comment:
don't call `createConfigFromMongoCLIConfig` instead of passing an error

Reply:
updated